Makefile

```diff
@@ -67,8 +67,14 @@ local:
 	@rm -rf /var/tmp/$(PKGNAME)-$(VERSION)
 	@echo "The archive is in $(PKGNAME)-$(VERSION).tar.gz"
 
+test-container:
+	sudo docker build -t larskarlitski/lorax-tests:latest -f Dockerfile.test .
```

Comment:
You don't need to change this, it isn't pushed to dockerhub and comes from your local system.
It would also be nice to be able to have it run this target if the image is missing. I'm not sure if there's a good way to express that in the Makefile though.

Reply:
I don't understand. One of the points of this PR is to use an image from dockerhub, so that
1. the builds on travis are faster with shorter logs
2. developers can download exactly the container that was used in testing to reproduce failures locally

Reply:
Right, but if it isn't on dockerhub it will use the local one. So to test it you don't need to change the name of it, you just need to build a local image with the same name.

Reply:
Ah, you mean to always use the local one (without `larskarlitski/`) in the Makefile and then pull it from dockerhub in `.travis.yml` before running `make test-in-docker`?

Reply:
No, what I mean is that if 'welder/lorax-tests:latest' doesn't exist in dockerhub it will look for a local image by that name. So you don't need (AFAIK) to do anything special other than push an image by that name. Basically, change it back to welder and it'll all get sorted out :)